Subscribers/Search.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace ShopwarePlugins\Connect\Subscribers;
+
+use Shopware\Components\Model\ModelManager;
+
+/**
+ * Class Property
+ * @package ShopwarePlugins\Connect\Subscribers
+ */
+class Search extends BaseSubscriber
+{
+
+    /**
+     * @var ModelManager
+     */
+    private $modelManager;
+
+    public function __construct(
+        ModelManager $modelManager
+    ) {
+        parent::__construct();
+        $this->modelManager = $modelManager;
+    }
+
+    public function getSubscribedEvents()
+    {
+        return array(
+            'Enlight_Controller_Action_PostDispatch_Backend_Search' => 'extendBackendPropertySearch',
+        );
+    }
+
+    /**
+     * @param \Enlight_Event_EventArgs $args
+     */
+    public function extendBackendPropertySearch(\Enlight_Event_EventArgs $args)
+    {
+        /** @var $subject \Enlight_Controller_Action */
+        $subject = $args->getSubject();
+        $request = $subject->Request();
+
+        switch ($request->getActionName()) {
+            case 'search':
+                $this->registerMyTemplateDir();
+                $this->registerMySnippets();
+
+                $entity = $request->getParam('entity', null);
+
+                switch ($entity) {
+                    case 'Shopware\Models\Property\Group':
```

Comment:
As @mkafadarov said please add use statements for Group, Option, Value and add Group::class instead of case 'Shopware\Models\Property\Group'. Same for other classes.